bokeh/server/session.py

```diff
@@ -165,8 +165,7 @@ def _wrap_session_callbacks(self, callbacks):
         return wrapped
 
     def _document_patched(self, event):
-        may_suppress = self._current_patch is not None and \
-                       self._current_patch.should_suppress_on_change(event)
+        may_suppress = getattr(event, 'setter_id', None) == self.id
```

Comment:
making events always have setter_id (but it can be None) is nicer than using getattr perhaps